Source/Core/VideoBackends/OGL/TextureCache.cpp

```diff
@@ -69,7 +69,8 @@ TextureCache::TextureCache()
     glBindTexture(GL_TEXTURE_BUFFER, m_palette_resolv_texture);
     glTexBuffer(GL_TEXTURE_BUFFER, GL_R16UI, m_palette_stream_buffer->m_buffer);
 
-    CreateTextureDecodingResources();
+    if (g_ActiveConfig.backend_info.bSupportsGPUTextureDecoding)
```

Comment:
Should add this check in the destructor as well, line 80.

Reply:
Done!